packages/shared/src/utils/index.ts

```diff
@@ -1,4 +1,6 @@
+export * from './ensure-extensions-dirs';
+export * from './generate-extensions-entry';
+export * from './get-extensions';
```

Comment:
Should these internal API things be in shared? Are they going to be used elsewhere?

Reply:
They are in `shared` to be usable in the API as well as in `vite.config.js`. I just split those into separate files and added `ensureExtensionsDirs()` to be able to init the extension folders in dev as well.